places/sandiego.md

```diff
@@ -0,0 +1,9 @@
+## San Diego, CA
+
+### Great Places to Eat
+
+ - Gami Sushi is an amazing restaurant
+
+### Fun things to do
+ 
+  - Surfing! Biking!
```

Comment:
@tomiy1 which are the other fun things to do other than :surfer: Surfing and :bicyclist:  Biking